doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst

```diff
@@ -93,12 +97,12 @@ with ``overrides.array_function_dispatch``:
 
 .. code:: python
 
-    def _asarray_decorator(a, dtype=None, order=None, like=None):
+    def _asarray_decorator(a, dtype=None, order=None, *, like=None):
         return (like,)
 
     @set_module('numpy')
     @array_function_dispatch(_asarray_decorator)
-    def asarray(a, dtype=None, order=None, like=None):
+    def asarray(a, dtype=None, order=None, *, like=None):
```

Comment:
There is a case for making `order` keyword only as well, but this is certainly sufficiently for now.

Reply:
IMO, while it's good to have things match for future reference, they're of lesser importance given this is just an example, I suspect these will anyway drift from future API changes.